packages/extension-chakra-storefront/src/pages/product-detail/partials/product-details.jsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2025, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import PropTypes from 'prop-types'
+import SimpleProductDetails from './product-details-simple'
+import CompositeProductDetails from './product-details-composite'
+
+const ProductDetails = (props) => {
+    const {isProductASet, isProductABundle} = props
+
+    if (isProductASet || isProductABundle) {
+        return <CompositeProductDetails {...props} />
+    }
+
+    return <SimpleProductDetails {...props} />
+}
```

Comment:
Minor: how about merging this back into the, um, _parent_ product details component? This isn't much code, so I think it doesn't need to be in its own component.